openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -122,3 +155,26 @@ func (configNodeV1) ValidateStatusUpdate(_ context.Context, uncastObj runtime.Ob
 
 	return errs
 }
+
+type validateCustomResourceWithClient struct {
```

Comment:
Nit: Name rotted during review?

```suggestion
type validateCustomResourceWithNodeLister struct {
```

Reply:
updated!